docs/content/en/docs/getting-started/local-environment/_index.md

```diff
@@ -71,11 +71,23 @@ The configuration specification is divided into two sections:
 
 For full EKS Anywhere configuration reference for a VMware vSphere cluster and explanation on each parameter in the configuration generated above refer vSphere configuration
 
+1. Generate a curated-packages config
+   {{% alert title="Note" color="primary" %}}
+   * It is *optional* to install the curated packages as part of the cluster creation and `eksctl anywhere` version should be `v0.9.0` or later.
+   * Post-creation installation and detailed package configurations can be found [here.]({{< relref "../../tasks/packages" >}})
+   {{% /alert %}}
+   Example shows how to install two packages `flux` and `harbor` from the [curated package list]({{< relref "../../reference/packagespec" >}}).
+   ```bash
+   eksctl anywhere generate package flux harbor -d .
+   ```
 
 1. Create a cluster
 
    ```bash
+   # Create a cluster without curated packages installation
    eksctl anywhere create cluster -f $CLUSTER_NAME.yaml
+   # Create a cluster with curated packages installation
+   eksctl anywhere create cluster -f $CLUSTER_NAME.yaml --install-packages ./curated-packages/
```

Comment:
Can't we use a default directory here. I think the packages directory would have to be pretty much hard coded for gitops